RocketBot2/Forms/MainForm.cs

```diff
@@ -292,13 +293,37 @@ private void GMAPSatellite_CheckedChanged(object sender, EventArgs e)
 
             SynchronizationContext.Post(o =>
             {
-                _playerOverlay.Markers.Clear();
                 if (_pokemonsOverlay.Markers.Count > 8)
                     _pokemonsOverlay.Markers.Clear();
-                _pokestopsOverlay.Routes.Clear();
-                _pokestopsOverlay.Markers.Clear();
-                _playerLocations.Clear();
 
+                try
+                {
+                    if (_session.Navigation.WalkStrategy.Points.Count > 0 && Points != _session.Navigation.WalkStrategy.Points)
```

Comment:
it is for load route point A to point B
![sans_titre](https://user-images.githubusercontent.com/11375035/28571055-4e1465a6-7141-11e7-9cd5-0205dccd49b0.png)